content/en/docs/setup/independent/high-availability.md

```diff
@@ -1,538 +1,522 @@
-* [Azure](https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-overview)
-
-You will need to ensure that the load balancer routes to **just `master0` on port 6443**. This is because kubeadm will perform health checks using the load balancer IP. Since `master0` is set up individually first, the other masters will not have running apiservers, which will result in kubeadm hanging indefinitely.
-
-If possible, use a smart load balancing algorithm like "least connections", and use health checks so unhealthy nodes can be removed from circulation. Most providers will provide these features.
-{{% /tab %}}
-{{% tab name="On-Site" %}}
-In an on-site environment there may not be a physical load balancer available. Instead, a virtual IP pointing to a healthy master node can be used. There are a number of solutions for this including keepalived, Pacemaker and probably many others, some with and some without load balancing.
-
-As an example we outline a simple setup based on keepalived. Depending on environment and requirements people may prefer different solutions. The configuration shown here provides an _active/passive_ failover without load balancing. If required, load balancing can by added quite easily by setting up HAProxy, NGINX or similar on the master nodes (not covered in this guide).
-
-1. Install keepalived, e.g. using your distribution's package manager. The configuration shown here works with version `1.3.5` but is expected to work with may other versions. Make sure to have it enabled (chkconfig, systemd, ...) so that it starts automatically when the respective node comes up.
-
-2. Create the following configuration file `/etc/keepalived/keepalived.conf` on all master nodes:
-     ```none
-     ! Configuration File for keepalived
-     global_defs {
-      router_id LVS_DEVEL
-     }
-
-     vrrp_script check_apiserver {
-      script "/etc/keepalived/check_apiserver.sh"
-      interval 3
-      weight -2
-      fall 10
-      rise 2
-     }
-
-     vrrp_instance VI_1 {
-        state <STATE>
-        interface <INTERFACE>
-        virtual_router_id 51
-        priority <PRIORITY>
-        authentication {
-            auth_type PASS
-            auth_pass 4be37dc3b4c90194d1600c483e10ad1d
-        }
-        virtual_ipaddress {
-            <VIRTUAL-IP>
-        }
-        track_script {
-            check_apiserver
-        }
-     }
-     ```
+      KUBECONFIG=/etc/kubernetes/admin.conf kubectl exec -n kube-system etcd-${CP0_HOSTNAME} -- etcdctl --ca-file /etc/kubernetes/pki/etcd/ca.crt --cert-file /etc/kubernetes/pki/etcd/peer.crt --key-file /etc/kubernetes/pki/etcd/peer.key --endpoints=https://${CP0_IP}:2379 member add ${CP1_HOSTNAME} https://${CP1_IP}:2380
+      kubeadm alpha phase etcd local --config kubeadm-config.yaml
+      ```
 
-    In the section `vrrp_instance VI_1`, change few lines depending on your setup:
+      - This command causes the etcd cluster to become unavailable for a
```

Comment:
I hate to say it, but I think the previous version was a bit better, this version doesn't really bring attention to the statement

Reply:
layout so ugly as to be distracting, though. I'll add it back in later.